plugin/storage/es/spanstore/dbmodel/to_domain_test.go

```diff
@@ -288,3 +288,34 @@ func TestFailureBadParentSpanID(t *testing.T) {
 	badParentSpanIDESSpan.ParentSpanID = "zz"
 	failingSpanTransformAnyMsg(t, &badParentSpanIDESSpan)
 }
+
+func CompareModelSpans(t *testing.T, expected *model.Span, actual *model.Span) {
+	model.SortSpan(expected)
+	model.SortSpan(actual)
+
+	if !assert.EqualValues(t, expected, actual) {
+		for _, err := range pretty.Diff(expected, actual) {
+			t.Log(err)
+		}
+		out, err := json.Marshal(actual)
+		assert.NoError(t, err)
+		t.Logf("Actual trace: %s", string(out))
+	}
+}
+func TestTagsMap(t *testing.T) {
+	tagsMap := map[string]interface{}{}
+	tagsMap["bool:bool"] = true
+	tagsMap["int.int"] = int64(1)
+	tagsMap["float"] = float64(1.1)
+	tagsMap["str"] = "foo"
+	tagsMap["binary"] = []byte("baz")
```

Comment:
Add a case for a negative test as well, like by passing a custom type (or just a `interface{}` type)